tests/kernel_performance/install_qatestset.pm

```diff
@@ -0,0 +1,52 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+#
+# Summary: setup performance run environment
+# Maintainer: Joyce Na <jna@suse.de>
+
+package install_qatestset;
+use base "y2logsstep";
+use power_action_utils 'power_action';
+use strict;
+use warnings;
+use utils;
+use testapi;
+
+sub install_pkg {
+    my $qa_server_repo = get_var('QA_REPO');
+    sleep 10;
+    assert_script_run("hostname > /etc/hostname");
```

Comment:
That's interesting. Why would we need that? Is the system wrongly configured by default?

Reply:
remove it